### PR TITLE
Image build patch

### PR DESCRIPTION
**Pull request by Ruilin-Ma**
Added content to the README.md to explain the scenario of switching instance types due to image build architecture.

### CODE REVIEW

**Comment by Ruilin-Ma** [comment]
Hi @dmuelle, would you please review the changes to the README file and let me know if there's anything I can help to enhance? Thanks!

File: README.adoc [hunk 1]
```diff
@@ -131,14 +131,18 @@ aws configure list
 
 === Provisioning a cluster
 
-The `eksctl` CLI tool greatly simplifies the process of creating clusters on EKS. To create your cluster, use the `eksctl create cluster` command:
+The `eksctl` CLI tool greatly simplifies the process of creating clusters on EKS. By default, the command includes a single `t2.small` Amazon Elastic Compute Cloud (EC2) instance that supports both `i386` and `x86_64` architectures. Please note that this instance is not covered under the AWS Free Tier; for more details, refer to the official https://aws.amazon.com/ec2/pricing/on-demand/[Amazon EC2 pricing^] documentation.
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
The `eksctl` CLI tool simplifies the process of creating clusters on EKS. By default, the command includes a single `t2.small` Amazon Elastic Compute Cloud (EC2) instance that supports both `i386` and `x86_64` architectures. However, this instance is not covered under the AWS Free Tier. For more information, see the official link:https://aws.amazon.com/ec2/pricing/on-demand/[Amazon EC2 pricing^] documentation.
```

File: README.adoc [hunk 2]
```diff
@@ -131,14 +131,18 @@ aws configure list
...
+
+If you need to build Docker images using the other architectures, such as `ARM64`, you will need to switch the instance type accordingly. To view the AWS supported instance types, please check out the navigation bar in EC2 home and navigate to ##Instances## > ##Instance Types##.
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
If you need to build Docker images using other architectures, such as `ARM64`, you must switch the instance type accordingly. To view the AWS supported instance types, use the navigation bar in EC2 home to select **Instances** > **Instance Types**. 
```

File: README.adoc [hunk 3]
```diff
@@ -131,14 +131,18 @@ aws configure list
...
+
+To create your cluster, use the `eksctl create cluster` command:
 
 [role=command]
 ```
 eksctl create cluster --name=guide-cluster --nodes=1 --node-type=t2.small
 ```
 
-Running this command creates a cluster that is called `guide-cluster` that uses a single `t2.small` Amazon Elastic Compute Cloud (EC2) instance as the worker node. The `t2.small` EC2 instance is not included in the AWS free tier. See the official https://aws.amazon.com/ec2/pricing/on-demand/[Amazon EC2 pricing^] documentation for more details. When the cluster is created, you see an output similar to the following:
+Running this command creates a cluster that is called `guide-cluster` that uses a single `t2.small` instance as the worker node. When the cluster is created, you will see an output similar to the following:
```
Comment (dmuelle) [inline README.adoc]:
```suggestion
Running this command creates a cluster that is called `guide-cluster` that uses a single `t2.small` instance as the worker node. When the cluster is created, you will see an output similar to the following example:
```

**Review by dmuelle — CHANGES_REQUESTED** [review]
ID review

**Comment by Ruilin-Ma** [comment]
Thanks @dmuelle for your review, your suggestions have merged into the PR. Please let me know if the current version looks good to you. Thanks!

**Review by dmuelle — APPROVED** [review]
LGTM, thanks!

**Pull request merged**